cypress/support/Actions.js

```diff
@@ -16,24 +16,35 @@ const helpers = require('../support/Helpers')
 export function CreateNewAction({ response, expectedEntities, requiredEntities, disqualifyingEntities, uncheckWaitForResponse, type = 'TEXT' }) {
   modelPage.NavigateToActions()
   actionModal.ClickNewAction()
-  // TODO: this is the default but we need to get this working... actionsModal.selectTypeText()
-  actionModal.TypeResponse(response)
-  actionModal.TypeExpectedEntity(expectedEntities)
-  actionModal.TypeRequiredEntities(requiredEntities)
-  actionModal.TypeDisqualifyingEntities(disqualifyingEntities)
+
+  // We do this first since we had a bug (1910) where it is not reset by the UI when
+  // type END_SESSION is selected.
   if (uncheckWaitForResponse) actionModal.UncheckWaitForResponse()
```

Comment:
I thought unchecking the wait before you select the END_SESSION type was the bug. This only works by taking advantage of that bug right?  I would think this should be placed after the `SelectType` and the test should fail if they attempt to uncheck an END_SESSION action.

Reply:
You are right that this is exercising that bug. I am intentionally doing the wrong thing so as to produce the bug. Right now, the only reason this test case passes is because of the code you see in the image below (part of this PR). Once dev fixes the bug in the UI, then I will uncomment that line of code and this test case will prove that the bug is fixed and does not regress.

![image](https://user-images.githubusercontent.com/15230446/52831804-d80f1d00-308a-11e9-833c-e0d76fc1bed8.png)